ccmlib/scylla_node.py

```diff
@@ -1224,12 +1228,22 @@ def get_node_scylla_version(self, scylla_exec_path=None):
 
         scylla_exec = os.path.join(scylla_exec_path, 'scylla')
 
-        scylla_version = subprocess.run(f"{scylla_exec} --version", shell=True, capture_output=True, text=True)
+        return subprocess.run(f"{scylla_exec} {option}", shell=True, capture_output=True, text=True)
+
+    def get_node_scylla_version(self, scylla_exec_path=None):
+        scylla_version = self.run_scylla_executable_with_option(option="--version", scylla_exec_path=scylla_exec_path)
         if scylla_version.returncode:
             raise NodeError("Failed to get Scylla version. Error:\n%s" % scylla_version.stderr)
 
         return scylla_version.stdout.strip()
 
+    def get_scylla_build_id(self):
```

Comment:
maybe we don't need this method? `scylla_build_id` property should be enough

Reply:
Good idea. Removed it